### DISCUSSION TITLE
I hope github support home screen iOS general notifications. So I can check my all notifications of github from homescreen

### DISCUSSION THREAD

**Discussion by SKbarbon** [community/community#58333] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.117.0 (101615344)
Device: iPhone 13 mini (iOS 17.0)
</details>

**Comment by iskae** [comment]
Hi @SKbarbon 👋 

Thanks a bunch for reaching out ❤️ We already have an internal tracking issue for this and we will try our best to bring this to our mobile apps 🎉 

Please stay tuned for our future versions 🙇 

Cheers!
